src/pages/AppDetail.svelte

```diff
@@ -118,8 +119,7 @@
         <ul>
           {#each filteredMetrics as metric}
             <li>
-              <a
-                href={`/apps/${params.app}/metrics/${metric.name}`}>{metric.name}</a>
+              <a href={getMetricURL(app.name, metric.name)}>{metric.name}</a>
```

Comment:
We do this dance because resources with a period (".") in them are considered "files with extensions" by the software we're using for serving static content locally ([sirv](https://github.com/lukeed/sirv))

Reply:
I'm not sure I follow this... the test example also has a period in it.